src/tests/models/fileManager.js

```diff
@@ -49,3 +53,30 @@ async function saveFile(fileUrl) {
     await database.saveFile(id, fileBase64Value);
     return id;
 }
+
+function validateJavascriptContent (javascriptFileContent) {
+    let error, errorMessage;
+    try {
+        const syntax = esprima.parseScript(javascriptFileContent, { tolerant: true });
+        const errors = syntax.errors;
+        if (errors.length > 0) {
+            let errorsString = '';
+            for (let i = 0; i < errors.length; i++) {
+                errorsString += errors[i].description + ', ';
+            }
+            errorsString = errorsString.substring(0, errorsString.length - 2);
```

Comment:
I am a fan of oneliners,
this loop can also be written like this
`errorsString = errors.map(error => error.description).join(', ')`

Reply:
ended up removing the errors array all-together